tests/stress_tests/common/utils.cpp

```diff
@@ -38,12 +38,56 @@ static int parseLine(std::string line) {
 
 #ifdef _WIN32
 size_t getVmSizeInKB() {
-                // TODO rewrite for Virtual Memory
-                PROCESS_MEMORY_COUNTERS pmc;
-                pmc.cb = sizeof(PROCESS_MEMORY_COUNTERS);
-                GetProcessMemoryInfo(GetCurrentProcess(),&pmc, pmc.cb);
-                return pmc.WorkingSetSize;
-	    }
+    PROCESS_MEMORY_COUNTERS pmc;
+    pmc.cb = sizeof(PROCESS_MEMORY_COUNTERS);
+    GetProcessMemoryInfo(GetCurrentProcess(),&pmc, pmc.cb);
+    return pmc.PagefileUsage / 1024;
+    }
+
+size_t getVmPeakInKB() {
+    PROCESS_MEMORY_COUNTERS pmc;
+    pmc.cb = sizeof(PROCESS_MEMORY_COUNTERS);
+    GetProcessMemoryInfo(GetCurrentProcess(),&pmc, pmc.cb);
```

Comment:
let's create static function for:
```
    PROCESS_MEMORY_COUNTERS pmc;
    pmc.cb = sizeof(PROCESS_MEMORY_COUNTERS);
    GetProcessMemoryInfo(GetCurrentProcess(),&pmc, pmc.cb);
```

And looks like `pmc` should be static too. It will prevent recreation of the object, but results still should be actual